### PR TITLE
[Wiki] Added guidelines to troubleshoot outdated Bicep versions

### PR DESCRIPTION
**Pull request by AlexanderSehr**
# Description

- Added guidelines to troubleshoot outdated Bicep versions

# Type of Change

Please delete options that are not relevant.

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Update to documentation

### CODE REVIEW

File: docs/wiki/The CI environment - Troubleshooting.md [hunk 1]
```diff
@@ -33,3 +33,21 @@ Potential reasons
...
+> - `Error BCP236: Expected a new line or comma character at this location.`
+> - `...` with the described reason below, there can be any amount of error messages you may encounter
+
+Potential reasons
+
+- The Bicep CLI version is outdated.
+  - **Solution:** Update the version of Bicep you're using on your machine. This can include:
+    - The Bicep extension for VS Code (update using, for example, the VSCode UI)
+    - The Bicep extension for the Azure CLI (update executing, for example, `az bicep upgrade`)
+    - The Bicep CLI (update executing, for example, `choco bicep upgrade`)
```
Comment (eriqua) [inline docs/wiki/The CI environment - Troubleshooting.md:52]: This won't work for those not using chocolatey, plus it's only relevant for Windows, let's copy the PS snippet or better reference the official docs page https://learn.microsoft.com/en-us/azure/azure-resource-manager/bicep/install#azure-powershell

Comment (AlexanderSehr) [reply docs/wiki/The CI environment - Troubleshooting.md:52]: Hence the "for example". Figured it would be better than nothing and gets the idea across

Comment (AlexanderSehr) [reply docs/wiki/The CI environment - Troubleshooting.md:52]: Can update it tomorrow

Comment (AlexanderSehr) [reply docs/wiki/The CI environment - Troubleshooting.md]: Updated and added a disclaimer to the front page as per some feedback I received

**Review by eriqua — APPROVED** [review]
🆗

**Pull request merged**